modules/usertracking-impl/src/main/java/org/opencastproject/usertracking/impl/UserSessionImpl.java

```diff
@@ -40,9 +41,11 @@
 
 @Entity(name = "UserSession")
 @Access(AccessType.FIELD)
-@Table(name = "oc_user_session")
+@Table(name = "oc_user_session", indexes = {
+    @Index(name = "IX_oc_user_session_user_id", columnList = "user_id")
```

Comment:
The field `user_id` is generated as `LONGTEXT` instead of `VARCHAR(128)`
Therefore the generation of the index throws the following exception:
`Internal Exception: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: BLOB/TEXT column 'user_id' used in key specification without a key length Error Code: 1170`

Reply:
Fixed